src/nvim/ops.c

```diff
@@ -5310,6 +5312,12 @@ static void free_register(struct yankreg *reg)
   y_current = curr;
 }
 
+int get_unnamed_register_name(void) {
+    int name = 0;
+    adjust_clipboard_name(&name, true, false);
```

Comment:
This function's style is off... we should probably use clang-format more as it's so much easier :smile: